test/end-to-end/core.sh

```diff
@@ -147,6 +148,11 @@ os::log::info "Ruby's testing blob digest: $rubyimageblob"
 os::log::info "Docker pullthrough"
 os::cmd::expect_success "oc import-image --confirm --from=mysql:latest mysql:pullthrough"
 os::cmd::expect_success "docker pull ${DOCKER_REGISTRY}/cache/mysql:pullthrough"
+mysqlblob="$(oc get istag -o go-template='{{range .image.dockerImageLayers}}{{if gt .size 1024.}}{{.name}},{{end}}{{end}}' "mysql:pullthrough" | cut -d , -f 1)"
```

Comment:
Make the size at least 4096. An empty layer pushed with docker 1.8 had at least 2000 bytes.